src/github/client/key-locator.test.ts

```diff
@@ -0,0 +1,68 @@
+import { keyLocator } from "~/src/github/client/key-locator";
+import { GitHubServerApp } from "~/src/models/github-server-app";
+import { existsSync, readFileSync } from "fs";
+import { mocked } from "ts-jest/utils";
+import { envVars } from "~/src/config/env";
+
+jest.mock("config/feature-flags");
+jest.mock("fs");
```

Comment:
IMO, you shouldn't mock FS as you're ignoring a _very_ important part of the keylocator which might break in staging/prod